wpilibNewCommands/src/main/native/include/frc2/command/CommandScheduler.h

```diff
@@ -209,30 +209,36 @@ class CommandScheduler final : public frc::Sendable,
   Command* GetDefaultCommand(const Subsystem* subsystem) const;
 
   /**
-   * Cancels a command.  The scheduler will only call the interrupted method of
-   * a canceled command, not the end method (though the interrupted method may
-   * itself call the end method).  Commands will be canceled even if they are
-   * not scheduled as interruptible.
+   * Cancels commands. The scheduler will only call {@link Command#end(boolean)}
+   * method of the canceled command with {@code true}, indicating they were
+   * canceled (as opposed to finishing normally).
```

Comment:
The documentation generator for C++ doesn't understand this link/code markup.  Just say `Command::End()` and `true`.